ngraph/python/tests/test_inference_engine/test_blob.py

```diff
@@ -72,66 +75,43 @@ def test_l():
 
 @pytest.mark.skip(reason="Need to figure out how to implement right conversion")
 def test_write_to_buffer_fp16():
-    tensor_desc = TensorDesc("FP16", [1, 3, 127, 127], "NCHW")
-    array = np.zeros(shape=(1, 3, 127, 127), dtype=np.float16)
-    blob = Blob(tensor_desc, array)
-    ones_arr = np.ones(shape=(1, 3, 127, 127), dtype=np.float16)
-    blob.buffer[:] = ones_arr
-    assert np.array_equal(blob.buffer, ones_arr)
+    write_to_buffer("FP16",np.float16)
```

Comment:
please, do the same: https://github.com/openvinotoolkit/openvino/pull/5020/files